live-examples/css-examples/positioned-layout/offsets.css

```diff
@@ -9,7 +9,7 @@
 
 #example-element {
     background-color: #ff7100;
-    border: 3px solid red;
+    border: 3px solid #00ff58;
```

Comment:
This change doesn't fix the contrast issue. Also, the green border and the orange background don't look good.

Reply:
Thanks, @NiedziolkaMichal. I make the changes